SPECS/readline/readline.spec

```diff
@@ -40,15 +45,17 @@ rm -rf %{buildroot}%{_infodir}
 %check
 make %{?_smp_mflags} check
 
-%post	-p /sbin/ldconfig
-%postun	-p /sbin/ldconfig
+%post   -p /sbin/ldconfig
+%postun -p /sbin/ldconfig
+
 %files
 %defattr(-,root,root)
 %license COPYING
-%{_libdir}/libreadline.so.7
-%{_libdir}/libhistory.so.7
-%{_libdir}/libhistory.so.7.0
-%{_libdir}/libreadline.so.7.0
+%{_libdir}/libreadline.so.8
```

Comment:
I think we might need to update all packages with `BuildRequires: readline` now.

Reply:
I don't think this is necessary for 2.0 since we've not yet released any packages. Let me know if there is another reason that would be required.

Reply:
That makes sense, you're right.